packages/victory-core/src/index.d.ts

```diff
@@ -249,13 +252,13 @@ export interface VictoryContainerProps {
   portalZIndex?: number;
   responsive?: boolean;
   style?: React.CSSProperties;
-  tabIndex?: number;
+  tabIndex?: NumberOrCallback;
```

Comment:
keep this as just number since it applies to the container?

Reply:
Hmm... I _think_ so. I'm having a hard time imagining a scenario where the tabIndex for the container might depend on props

Reply:
for sure, I was torn between keeping the tabIndex type consistent throughout the app since it does cover just `number` , or using the type to better illustrate how the prop can/should be applied, of which the latter seems more correct

Reply:
changed back to `number`